### PR TITLE
Runtime corpus test now uses more interesting snapshot times

### PR DESCRIPTION
**Pull request by jystic**
I also made the avalanche pretty printing a bit more compact / easier to read.

! @tranma

### CODE REVIEW

**Comment by jystic** [comment]
! @amosr seeing as you're here

File: icicle-compiler/test/Icicle/Test/Runtime/Corpus.hs [hunk 1]
```diff
@@ -121,12 +121,16 @@ handleDiscard :: MonadCatch m => PropertyT m a -> PropertyT m a
...
+  unsafeTimeOfYMD 3000 1 1
+
 test_runtime_corpus :: CorpusId -> (InputName, OutputId, Text) -> Property
 test_runtime_corpus cid query =
   property . handleDiscard $ do
-    stime <- forAll $ Gen.arbitrary
-    maxMapSize <- forAll $ Gen.int (Range.linear 1 100)
     wt <- forAll . Gen.quickcheck $ genWellTypedFromSource query
+    maxMapSize <- forAll $ Gen.int (Range.linear 1 100)
+    stime <- forAll $ Gen.element (timeEnd : fmap atTime (concat . Map.elems $ wtInputs wt))
```
Comment (amosr) [inline icicle-compiler/test/Icicle/Test/Runtime/Corpus.hs]: funny thing is, I think `inputsForType` generates a date too, but `genWellTypedFromSource` is throwing it away

**Comment by amosr** [comment]
pretty good 👍

File: icicle-compiler/test/Icicle/Test/Runtime/Corpus.hs (hunk 1 above)
Comment (jystic) [inline icicle-compiler/test/Icicle/Test/Runtime/Corpus.hs]: 😆

Comment (tranma) [inline icicle-compiler/test/Icicle/Test/Runtime/Corpus.hs]: yeah, that was deliberate when I was generating multiple `Cluster`

**Pull request merged**